x-pack/platform/test/fleet_api_integration/apis/integrations/inputs_with_standalone_docker_agent.ts

```diff
@@ -5,18 +5,40 @@
  * 2.0.
  */
 
+import axios from 'axios';
+import { last } from 'lodash';
 import { ToolingLog } from '@kbn/tooling-log';
 import { tmpdir } from 'os';
 import { writeFile } from 'fs/promises';
 import { v4 as uuid } from 'uuid';
 import execa from 'execa';
 import path from 'path';
 import expect from '@kbn/expect';
-
+import { retryForSuccess } from '@kbn/ftr-common-functional-services';
 import { FtrProviderContext } from '../../../api_integration/ftr_provider_context';
-import { getLatestVersion } from '../../../fleet_cypress/artifact_manager';
 import { skipIfNoDockerRegistry } from '../../helpers';
 
+const DEFAULT_VERSION = '8.15.0-SNAPSHOT';
+const name = 'Artifact Manager - getLatestVersion';
+
+export async function getLatestVersion(): Promise<string> {
+  return retryForSuccess(
+    new ToolingLog({ level: 'debug', writeTo: process.stdout }, { context: name }),
+    {
+      timeout: 60_000,
+      methodName: name,
+      retryCount: 20,
+      block: () => axios('https://artifacts-api.elastic.co/v1/versions'),
+    }
+  )
+    .then(
+      (response) =>
+        last((response.data.versions as string[]).filter((v) => v.includes('-SNAPSHOT'))) ||
+        DEFAULT_VERSION
+    )
+    .catch(() => DEFAULT_VERSION);
+}
```

Comment:
moved here to avoid import from `x-pack/test` as it will bring circular dependency b/w test-helper packages